lib/ansible/cli/galaxy.py

```diff
@@ -469,11 +469,34 @@ def add_install_options(self, parser, parents=None):
                                  'collection. This will not ignore dependency conflict errors.'
         else:
             args_kwargs['help'] = 'Role name, URL or tar file'
+            if self._implicit_role:
+                args_kwargs['help'] += '. This is mutually exclusive with --requirements-file.'
+            else:
+                args_kwargs['help'] += '. This is mutually exclusive with --role-file.'
             ignore_errors_help = 'Ignore errors and continue with the next specified role.'
 
+        if self._implicit_role:
+            # might install both roles and collections
+            description_text = (
+                'Install roles and collections from file(s), URL(s) or Ansible '
+                'Galaxy to the first entry in the config COLLECTIONS_PATH for collections '
+                'and first entry in the config ROLES_PATH for roles. '
+                'The first entry in the config ROLES_PATH can be overridden by --roles-path '
+                'or -p. This will result in only roles being installed.'
```

Comment:
```suggestion
                'or -p, but this will result in only roles being installed.'
```

Reply:
@s-hertel I read and reread this one a dozen times to see if there is any clearer way to say this, but I don't think there is. I have a minor suggestion here, but it doesn't really clarify it any, so take it or leave it so to speak.

Reply:
@samccann Thanks for the second set of eyes!